### ISSUE TITLE
remove Save & Grade button

### ISSUE THREAD

**Issue by mfsilva22** [PrairieLearn/PrairieLearn#3841]
Some "questions" are not actually graded (video lectures, workspace, etc). It makes more sense to not have a "Save & Grade" button for these cases.

Can we just use the `gradingMethod` property for this? Maybe `"gradingMethod" : None`  does not display any button? 

I am not sure about the direction for Manual Grading, but potentially some questions will only have the button as "Save", which could also be controlled by the same property.

**Comment by firasm** [comment]
Hi @mfsilva22 - thanks for creating an issue on this! Jonatan pointed me to this issue and the feature would be useful for me as well!

For me, two use cases:

1) I like to add informational "sign posts" for natural breaks in my assignments/tests, and a way to remove the Save and Grade button, or to have a "blank" element that is not graded would be useful.

2) For in-class activities, I'm toying with the idea of embedding in Sli.do or PollEV polls in PL elements - removing the save and grade button here would be useful :-)

<img width="1067" alt="screen_shot_2021-08-31_at_10 09 43_am" src="https://user-images.githubusercontent.com/2507459/131559249-bba1e12f-5e63-458a-b897-47bbebb20435.png">
<img width="1067" alt="screen_shot_2021-08-31_at_10 09 48_am" src="https://user-images.githubusercontent.com/2507459/131559269-4f37c9f6-5d50-4eb8-bd0e-83f1dbacf566.png">

**Comment by trombonekenny** [comment]
I'm running across this now in my classes. A `gradingMethod: "None"` doesn't seem too hard to implement.

**Comment by trombonekenny** [comment]
Specifically, I use a custom element that lets them run SQL queries in Javascript and I have questions that are "sandboxes" that aren't graded but give them question panel information and a place to try their queries. In homeworks, students get unlimited submissions but in my exams, "points: 0" only gives a single attempt and then makes the question read-only.

`gradingMethod: Manual` but that's still misleading as we're never going to grade them.

**Comment by nwalters512** [comment]
I agree that it would be useful to have questions that exist just to display information or provide a sandbox. Some questions to drive the discussion and a potential implementation plan:

- Should it be possible to submit to this kind of question (though without running `grade()`)?
  - If submissions are allowed, is this distinct from a question that doesn't even have a submit button?
  - If submissions are allowed, should we run `parse()`? We'd probably need to so that elements can render correctly in the submission panel.
- Will we have to change the student/instructor assessment instance pages to indicate that these questions don't have points/scores/etc?
- How do we communicate to students that a question doesn't accept submissions, or that submissions won't be graded? This may be important for CBTF use cases where proctors might not have full context on the exam, as students will inevitably get confused if they can submit to some questions but not others, even if it might be entirely obvious from actually reading the question.
- Does this interact with #6262 at all? I believe there was talk of deprecating `gradingMethod` entirely, though I couldn't immediately find that anywhere in writing.

**Comment by jonatanschroeder** [comment]
For reference:

* Canvas quizzes has "Text question" that have no submittable component: https://community.canvaslms.com/t5/Instructor-Guide/How-do-I-create-a-Text-no-question-quiz-question/ta-p/1028
* Moodle uses "Description question": https://docs.moodle.org/404/en/Description_question_type

**Comment by reteps** [comment]
Related to https://github.com/PrairieLearn/PrairieLearn/issues/11560

**Comment by jonatanschroeder** [comment]
> Related to [#11560](https://github.com/PrairieLearn/PrairieLearn/issues/11560)

Can you elaborate? I believe this is a different problem.

**Comment by reteps** [comment]
> > Related to [#11560](https://github.com/PrairieLearn/PrairieLearn/issues/11560)
> 
> Can you elaborate? I believe this is a different problem.

Both are related to skipping grading; this is at the question level, whereas #11560 is at the element level. This PR also requires additional UI on-top of skipping grading. You are right, they are not the same problem.

**Comment by nwalters512** [comment]
Yeah, please think before adding crosslinks everywhere. This is unrelated. Hiding the above comments as off-topic.

**Comment by reteps** [comment]
Related: #3345

**Comment by jonatanschroeder** [comment]
I don't think they're the same thing. This one is for question authors to identify that the question has not submittable content. The other is a student UX   for something similar to disabling real-time grading. 

Also note that this is intended to disable both buttons (Save&Grade and Save Only), while the other one only disables the grade button.
